src/Commands/TankDrive.h

```diff
@@ -21,7 +22,7 @@
 class TankDrive: public frc::Command {
```

Comment:
`TankDrive` needs a method to set `mAutomaticShifting`. This method would be called by `ToggleAutoShifting.execute()`.